WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridFragment.java

```diff
@@ -630,20 +626,6 @@ public void onClick(DialogInterface dialog, int which) {
         dialog.show();
     }
 
-    private void handleMultiSelectPost() {
-        if (!isAdded()) {
-            return;
-        }
-
-        ArrayList<Long> remoteMediaIds = new ArrayList<>();
-        for (int localMediaId : mGridAdapter.getSelectedItems()) {
-            MediaModel mediaModel = mMediaStore.getMediaWithLocalId(localMediaId);
-            remoteMediaIds.add(mediaModel.getMediaId());
-        }
-
-        ActivityLauncher.newGalleryPost(getActivity(), mSite, remoteMediaIds);
```

Comment:
This leaves `ActivityLauncher.newGalleryPost()` unused - I think we should probably clean that up too.